From randy.dunlap@oracle.com  Fri Jun  5 11:15:53 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Fri, 05 Jun 2009 11:10:42 -0700
Subject: Staging: comedi: uses udelay, needs delay.h
To: <devel@driverdev.osuosl.org>
Cc: Greg KH <gregkh@suse.de>
Message-ID: <4A295FA2.1060401@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

comedi driver(s) use udelay() so they need to #include delay.h.

drivers/staging/comedi/drivers/adq12b.c: In function 'adq12b_ai_rinsn':
drivers/staging/comedi/drivers/adq12b.c:328: error: implicit declaration of function 'udelay'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/comedidev.h |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/staging/comedi/comedidev.h
+++ b/drivers/staging/comedi/comedidev.h
@@ -28,6 +28,7 @@
 #include <linux/module.h>
 #include <linux/kdev_t.h>
 #include <linux/slab.h>
+#include <linux/delay.h>
 #include <linux/errno.h>
 #include <linux/spinlock.h>
 #include <linux/mutex.h>
